data_generation/define_experiment.py

```diff
@@ -304,12 +304,17 @@ def get_defines_list(var, ent, identity=True, defn_type='is_isnt', top_k=10):
 
     # sort definitions and QA test sets by entity (needed for gradient alignment experiments) 
     for subset_name in defns:
-        defns[subset_name] = sorted(defns[subset_name], key=lambda x: x.entity)
+        defns[subset_name] = sorted(defns[subset_name], key=lambda x: x.variable)
     for subset_name in qa_test_sets:
-        qa_test_sets[subset_name] = sorted(qa_test_sets[subset_name], key=lambda x: x.question.entity)
+        if subset_name == 'q_no_replacement_baseline':
```

Comment:
might be worth it to unite the for loops on lines 308 and 313? they do the same thing

Reply:
They are over different subsets. Why?